aiogram/dispatcher/filters/builtin.py

```diff
@@ -12,6 +12,19 @@
 from aiogram.types import CallbackQuery, Message, InlineQuery, Poll, ChatType
 
 
+IDFilterArgumentType = typing.Union[typing.Iterable[typing.Union[int, str]], str, int]
+
+
+def extract_filter_ids(id_filter_argument: IDFilterArgumentType) -> typing.Set[int]:
```

Comment:
I think it would be better to put this method into `IDFilter` as `@staticmethod`

Reply:
Should it be unique to IDFilter though?
You may use the same type & helper function in other filters.

This can be used in AdminFilter with a small changes.

Reply:
Hmm.. yes, it can.

Reply:
also, for me, the naming is a bit obscure. `extract_filter_ids` says about `filter_ids` like about a special type of `ids`. As these are just regular chat ids, I would think about changing the name to `extract_chat_ids`, but that's up to you

Reply:
> shouldn't we consider moving it to `utils`

@Birdi7 I think no as it's internal helper for this file (for now).

Reply:
this function also can be useful for users of lib in some cases, but ok